fix(tegel): remove base value and use px instead of rem (#691)
scania-digital-design-system/sdds: tegel/src/stories/Migration/Migration.stories.mdx

# Migration Docs

Here is where we document all the breaking changes that we are introducing while migrating SDDS to Tegel.
<!-- TODO: remove link to markdownguide.org -->
<!-- ### MD cheatsheet
https://www.markdownguide.org/cheat-sheet/ -->

---

## Removed base font-size and moved to `px` instead of `rem`.

This change affects all components/typography/spacing/grid. In the past we have used a base value of 4px
that has been used to calculate the spacing/padding/margins etc with rem. This was done by setting the 
base value (4px) as the font-size on the `<html>`-element, which for some users has caused error with 
custom font-sizes. We have now removed these dependancies and are using pixels in our 
components/spacing/typography/margins.

We are doing this to decrease the dependencies between our packages and we want to make sure the Tegel
can be used on it's own. 

> Ideally this change should not affect you, but since it is something that has an affect on all components

Reviewer: Maybe instead: "If you are not using rem yourself, this change should not affect you".

And we could also write something about what to do if they are using rems

Author: Yeah, nice!

# Migration Docs

Here is where we document all the breaking changes that we are introducing while migrating SDDS to Tegel.
<!-- TODO: remove link to markdownguide.org -->
<!-- ### MD cheatsheet
https://www.markdownguide.org/cheat-sheet/ -->

---

## Removed base font-size and moved to `px` instead of `rem`.

This change affects all components/typography/spacing/grid. In the past we have used a base value of 4px
that has been used to calculate the spacing/padding/margins etc with rem. This was done by setting the 
base value (4px) as the font-size on the `<html>`-element, which for some users has caused error with 
custom font-sizes. We have now removed these dependancies and are using pixels in our 
components/spacing/typography/margins.

We are doing this to decrease the dependencies between our packages and we want to make sure the Tegel
can be used on it's own. 

> ## What action is required? 

Author: ```suggestion
#### What action is required? 
```

> ## Removed base font-size and moved to `px` instead of `rem`.
> 
> This change affects all components/typography/spacing/grid. In the past we have used a base value of 4px
> that has been used to calculate the spacing/padding/margins etc with rem. This was done by setting the 
> base value (4px) as the font-size on the `<html>`-element, which for some users has caused error with 
> custom font-sizes. We have now removed these dependancies and are using pixels in our 
> components/spacing/typography/margins.
> 
> We are doing this to decrease the dependencies between our packages and we want to make sure the Tegel
> can be used on it's own. 
> 
> #### What action is required? 
> 
> If you are not yourself using `rem` your project ideally this change should not affect you, but since it is 
> something that has an affect on all components and utitlies/foundations we expect that there might be some
> unforseen problems. These will have our highest prio in order to make the migration to Tegel as smooth as possible.
> 
> If you are however using `rem` in your project you would have to consider that the base value will no longer be `4px`.
> This value will now by default be `16px`. So for the element in you project that are using rem these would have to be 
> recalculated.

Reviewer: ```suggestion
## Removed base font-size and moved to `px` instead of `rem`.

This change affects all components/typography/spacing/grid. 

In the past we have used a base value of 4px
that has been used to calculate the spacing/padding/margins etc with rem. This was done by setting the 
base value (4px) as the font-size on the `<html>`-element, which for some users has caused error with 
custom font-sizes. We have now removed these dependencies and are using pixels in our 
components/spacing/typography/margins.

We are doing this to decrease the dependencies between our packages and we want to make sure the Tegel
can be used on its own. 

#### What action is required? 

If you are not yourself using `rem` your project ideally this change should not affect you, but since it is 
something that has an effect on all components and utilities/foundations we expect that there might be some
unforeseen problems. These will have our highest prio in order to make the migration to Tegel as smooth as possible.

If you are however using `rem` in your project you would have to consider that the base value will no longer be `4px`.
This value will now by default be `16px`. So for the element in you project that are using rem these would have to be 
recalculated.
```

# Migration Docs

Here is where we document all the breaking changes that we are introducing while migrating SDDS to Tegel.
<!-- TODO: remove link to markdownguide.org -->
<!-- ### MD cheatsheet
https://www.markdownguide.org/cheat-sheet/ -->

---

## Removed base font-size and moved to `px` instead of `rem`.

This change affects all components/typography/spacing/grid. 

In the past we have used a base value of 4px
that has been used to calculate the spacing/padding/margins etc with rem. This was done by setting the 
base value (4px) as the font-size on the `<html>`-element, which for some users has caused error with 
custom font-sizes. We have now removed these dependencies and are using pixels in our 
components/spacing/typography/margins.

We are doing this to decrease the dependencies between our packages and we want to make sure the Tegel
can be used on its own. 

#### What action is required? 

> If you are not yourself using `rem` your project ideally this change should not affect you, but since it is 

Author: ```suggestion
If you are not yourself using `rem` in your project ideally this change should not affect you, but since it is 
```
